webrtc.html

```diff
@@ -7039,6 +7044,63 @@ <h2>Attributes</h2>
               <a>[[\ReceiverRtcpTransport]]</a> slot.</p>
             </dd>
           </dl>
+          <dt><code>jitterBufferDelayHint</code> of type <span class=
+          "idlAttrType"><a>double</a></span>, nullable</dt>
+          <dd>
+            <p>This attribute gives the application the ability to affect the
+            length of the jitter buffer using a <i>hint</i>. This allows the
+            application to influence the trafeoffs between having a longer or
+            shorter jitter buffer. The longer the buffer, the more chance there
+            is that packets arrive in time for playout, but the higher the delay
+            is. The shorter the buffer, the higher the risk of stuttering, but
+            the shorter the playout delay. For interactive applications, low
+            delay may be important, wheras for passive applications a
+            stutter-free experience may be more important.</p>
+            <p>Note that the delay is only a hint; a soft target. The underlying
+            congestion control mechanisms employed by user agensts are outside
```

Comment:
Congestion control is not the same as jitter buffer delay.

Reply:
playout buffering mechanisms? (stolen from the perkins book)